### PR TITLE
Added listener on aws bedrock

### PR DESCRIPTION
**Pull request by Claudio-code**
## Issue

Closes #

issue https://github.com/langchain4j/langchain4j/issues/1759

## General checklist

- [x] There are no breaking changes
- [x] I have added unit and integration tests for my change
- [x] I have manually run all the unit and integration tests in the module I have added/changed, and they are all green
- [x] I have manually run all the unit and integration tests in the [core](https://github.com/langchain4j/langchain4j/tree/main/langchain4j-core) and [main](https://github.com/langchain4j/langchain4j/tree/main/langchain4j) modules, and they are all green

- [ ] I have added/updated the [documentation](https://github.com/langchain4j/langchain4j/tree/main/docs/docs)
- [ ] I have added an example in the [examples repo](https://github.com/langchain4j/langchain4j-examples) (only for "big" features)
- [ ] I have added/updated [Spring Boot starter(s)](https://github.com/langchain4j/langchain4j-spring) (if applicable)

## Checklist for adding new model integration

- [ ] I have added my new module in the [BOM](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-bom/pom.xml)

## Checklist for adding new embedding store integration

- [ ] I have added a `{NameOfIntegration}EmbeddingStoreIT` that extends from either `EmbeddingStoreIT` or `EmbeddingStoreWithFilteringIT`
- [ ] I have added my new module in the [BOM](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-bom/pom.xml)

## Checklist for changing existing embedding store integration

- [ ] I have manually verified that the `{NameOfIntegration}EmbeddingStore` works correctly with the data persisted using the latest released version of LangChain4j

### CODE REVIEW

**Comment by Claudio-code** [comment]
Passed in all tests, except 2 because I not can acess model

image
image

File: langchain4j-bedrock/src/test/java/dev/langchain4j/model/bedrock/BedrockChatModelListenerIT.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+import java.time.Duration;
+import java.util.Collections;
+
+public class BedrockChatModelListenerIT extends ChatModelListenerIT {
+    @Override
+    protected ChatLanguageModel createModel(ChatModelListener listener) {
+        return BedrockAnthropicMessageChatModel
+                .builder()
+                .temperature(0.50f)
```
Comment (langchain4j) [inline langchain4j-bedrock/src/test/java/dev/langchain4j/model/bedrock/BedrockChatModelListenerIT.java:16]: no need to override the value for temperature and other parameters, please check how other similar tests are implemented

**Review by ghost — APPROVED** [review]
@Claudio-code thank you!

**Comment by langchain4j** [comment]
Hi @Claudio-code and @herbert-beckman, do you use AWS Bedrock models? Do they work for you? If yes, which settings are you using (model id, region, etc)? Thanks!

(There seems to be some issue with my account as I can't access any Bedrock models...)

**Comment by herbert-beckman** [comment]
Hi @langchain4j. Yes, I do. My authentication settings are in the credentials file. I only have access to a few models (cohere, anthropic, llama3, mistral, etc). My organization configures these accesses. I use AZ us-east-1 and this [model ids](https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html).

image

**Comment by langchain4j** [comment]
@herbert-beckman thanks a lot for the info!

**Pull request merged**